AsyncDisplayKit/ASViewController.h

```diff
@@ -9,14 +9,31 @@
 #import <UIKit/UIKit.h>
 #import <AsyncDisplayKit/ASDisplayNode.h>
 
+@class ASTraitCollection;
+
 NS_ASSUME_NONNULL_BEGIN
 
+typedef ASTraitCollection * _Nonnull (^ASDisplayTraitsForTraitCollectionBlock)(UITraitCollection *traitCollection);
+typedef ASTraitCollection * _Nonnull (^ASDisplayTraitsForTraitWindowSizeBlock)(CGSize windowSize);
+
 @interface ASViewController<__covariant DisplayNodeType : ASDisplayNode *> : UIViewController
 
 - (instancetype)initWithNode:(DisplayNodeType)node NS_DESIGNATED_INITIALIZER;
 
 @property (nonatomic, strong, readonly) DisplayNodeType node;
 
+@property (nonatomic, strong) id displayTraitsContext;
```

Comment:
The documentation of this property should be very clear — its usefulness and also the internal memory behavior.